securedrop/bin/update-requirements

```diff
@@ -2,23 +2,29 @@
 # shellcheck disable=SC2086
 set -euxo pipefail
 
-COMMON_ARGS="--generate-hashes --python-version 3.8 --python-platform x86_64-unknown-linux-gnu --custom-compile-command ./securedrop/bin/update-requirements"
+# Our $UBUNTU_VERSION is not available at runtime.
+source /etc/os-release
+
+# https://peps.python.org/pep-0508/#environment-markers
+PYTHON_VERSION="$(python3 -c 'import platform; print(".".join(platform.python_version_tuple()[:2]))')"
+
+COMMON_ARGS="--generate-hashes --python-version ${PYTHON_VERSION} --python-platform x86_64-unknown-linux-gnu --custom-compile-command ./securedrop/bin/update-requirements"
 
 cd "$(git rev-parse --show-toplevel)/securedrop"
 
 uv pip compile ${COMMON_ARGS} \
-    --output-file requirements/python3/develop-requirements.txt \
-    requirements/python3/translation-requirements.in \
-    requirements/python3/develop-requirements.in
+    --output-file requirements/${VERSION_CODENAME}/develop-requirements.txt \
+    requirements/translation-requirements.in \
```

Comment:
`make update-python3-requirements` fails, as .in files are still in `.../python3/`

Reply:
Oops; missed that in doing the refactoring @legoktm suggested.  Fixed in ff0b8acaea8ad2ea69dfab170ef3e2c02bd9c55f; diff confirms that (apart from a side effect of rebasing from `develop`) just paths changed even after rerunning `make update-python3-requirements`:

https://github.com/freedomofpress/securedrop/compare/c11b76d5f922ea61ced8f22fca309f71e2bbe992..ff0b8acaea8ad2ea69dfab170ef3e2c02bd9c55f